resources/functionMap.php

```diff
@@ -6714,161 +6714,170 @@
 'MongoDB::setReadPreference' => ['bool', 'read_preference'=>'string', 'tags='=>'array'],
 'MongoDB::setSlaveOkay' => ['bool', 'ok='=>'bool'],
 'MongoDB::setWriteConcern' => ['bool', 'w'=>'mixed', 'wtimeout='=>'int'],
-'MongoDB\BSON\fromJSON' => ['string', 'json'=>'string'],
-'MongoDB\BSON\fromPHP' => ['string', 'value'=>'object|array'],
-'MongoDB\BSON\toCanonicalExtendedJSON' => ['string', 'bson'=>'string'],
-'MongoDB\BSON\toJSON' => ['string', 'bson'=>'string'],
-'MongoDB\BSON\toPHP' => ['object|array', 'bson'=>'string', 'typemap='=>'?array'],
-'MongoDB\BSON\toRelaxedExtendedJSON' => ['string', 'bson'=>'string'],
-'MongoDB\Driver\Monitoring\addSubscriber' => ['void', 'subscriber'=>'MongoDB\Driver\Monitoring\Subscriber'],
-'MongoDB\Driver\Monitoring\removeSubscriber' => ['void', 'subscriber'=>'MongoDB\Driver\Monitoring\Subscriber'],
-'MongoDB\BSON\Binary::__construct' => ['void', 'data'=>'string', 'type='=>'int'],
+'MongoDB\BSON\fromJSON' => ['string', 'json' => 'string'],
```

Comment:
Because of the `=>` and ` => ` change this is very hard to review. Please do not change this formatting as the rest of the file also use `=>`.